cmd/mirror-url.go

```diff
@@ -141,8 +149,7 @@ func deltaSourceTarget(sourceURL string, targetURL string, isForce bool, isFake
 			}
 		case differInSecond:
 			if isRemove {
-				// todo(nl5887): I'd all force and fake checks to the the actual mirror / harvest
-				if !isForce && !isFake {
+				if !isFake {
```

Comment:
Something weird here,

`isRemove` is set and we are throwing an error errDeleteNotAllowed if `isFake` is not enabled ?

Reply:
Logic is not correct.